src/blockcholesky.jl

```diff
@@ -0,0 +1,27 @@
+
+# Function for cholesky decomposition on block matries
+# 'cholesky' is the build-in one in LAPACK
+
+function Bcholesky(A::Symmetric{<:Any,<:BlockArray})
+    chol_U = BlockArray{Float64}(zeros(size(A)), fill(size(A)[1]÷blocksize(A)[1], blocksize(A)[1]), fill(size(A)[1]÷blocksize(A)[1], blocksize(A)[1]))
+
+    # Initializing the first role of blocks
+    chol_U[Block(1,1)] = cholesky!(A[Block(1,1)]).U
```

Comment:
When we change this to be properly in-place, we will want to replace this with `cholesky!(Symmetric(getblock(parent(A),1,1)))` which will modify `A`.